client/src/hooks/journals/use-select-volume/index.js

```diff
@@ -2,11 +2,11 @@ import { useSelector } from "react";
 import { fixtures } from "helpers/storybook/exports";
 
 // Redo this hook (or don't use if all volume data is contained in the journal) with api
-export default function useSelectVolume(journal, slug) {
-  const volume = journal
-    ? journal.relationships.volumes.filter(
-        item => item.attributes.slug === slug
-      )[0]
-    : null;
-  return { volume };
+export default function useSelectVolume(match, journal) {
+  if (!journal) return { volume: null, volumeResponse: null };
+
+  const volume = journal.relationships.volumes.find(
+    v => v.id === match.params.volumeSlug
+  );
```

Comment:
Thank you for this! How did I not know about `.find()`??

Reply:
🙂 

It's hard to keep track of all the Array methods in JS. I have to go looking for them all the time.